src/EditorFeatures/CSharpTest2/Recommendations/MethodKeywordRecommenderTests.cs

```diff
@@ -196,5 +196,16 @@ public async Task TestNotInEnum()
 @"enum E {
     [$$");
         }
+
+        [Fact]
+        public async Task TestPrimaryConstructor()
+        {
+            await VerifyKeywordAsync("""
+                [$$
+                class C()
+                {
+                }
+                """);
+        }
```

Comment:
```suggestion
        [Fact]
        public async Task TestClassPrimaryConstructor()
        {
            await VerifyKeywordAsync("""
                [$$
                class C()
                {
                }
                """);
        }

        [Fact]
        public async Task TestRecordPrimaryConstructor()
        {
            await VerifyKeywordAsync("""
                [$$
                record C()
                {
                }
                """);
        }
```

Reply:
This is exactly the sort of jail-breaking-into I was hoping to avoid. Is `method:` even supported for records? They didn't for me when I tried quickly in SharpLab

Reply:
They used to be a warning. But when primary constructors were implemented, it seems they became supported, and now they works on "Preview" and produces an error on C# 11 or lower:

![image](https://github.com/dotnet/roslyn/assets/31348972/7d04ebff-4e2c-4c76-bc05-014b4511f30b)

Reply:
The jail breaking continues.. 

This is a class declaration, with an attribute list
```cs
[
class C
{
}
```

This is a compilation unit, with an EndOfFileToken, and a bunch of skipped token trivia
```cs
[
record C
{
}
```

🤦‍♂️